addons/purchase_requisition/static/src/widgets/purchase_order_alternatives_widget.js

```diff
@@ -16,11 +16,10 @@ FieldMany2ManyAltPOsRenderer.recordRowTemplate = "purchase_requisition.AltPOsLis
 
 export class FieldMany2ManyAltPOs extends X2ManyField {
    setup() {
+      super.setup();
       this.orm = useService("orm");
       this.action = useService("action");
-      // TODO: this is a terrible hack, make this a proper extension of many2many if/when possible
-      this.props.record.activeFields[this.props.name].widget = "many2many";
-      super.setup();
```

Comment:
If I remember correctly, I purposely set the `.widget = "many2many"` before the super.setup() because the setup() depends on the value being forced to many2many for the link feature to work correctly. I.e. it will throw an error when you try to link additional POs (video from the PR's runbot): 
[Odoo---Requests-for-Quotation.webm](https://user-images.githubusercontent.com/57942143/221884787-083f7f33-d5e6-41f4-a18d-b46550eb43ee.webm)

So I guess you only need to add `this.isMany2Many = true;` before the setup()? (I didn't test)